config/initializers/secure_headers.rb

```diff
@@ -44,8 +44,13 @@
   config.cookies = {
     secure: true, # mark all cookies as "Secure"
     httponly: true, # mark all cookies as "HttpOnly"
+    # We need to set the SameSite setting to "Lax", not "Strict" due to a bug
+    # in Chrome that resets the session in the new browser tab that opens when
+    # the email confirmation link is clicked. Resetting the session means losing
+    # all the SP info we stored there, meaning during account creation, a user
```

Comment:
Yes, we would need to change the point in time where we update the session from `EmailConfirmationsController` to `PasswordsController`. What's happening is that Chrome is undoing what we set after the confirmation link is clicked. Since we still include the `request_id` on the `/sign_up/enter_password` page, we can set the session once the password is entered.  

For now, this is the easiest fix, given that the RC has pulled out my changes from #1265.

Some of us have spent most of the day tracking down this issue that was reported by our partner agencies, and we thought it might have been due to my changes, but it turns out it was a Chrome bug that was exposed due to our change of this cookie setting.

Even though my code probably works fine, we feel it's safer to launch without it, then have more time to test it thoroughly before deploying it.